mlflow/store/file_store.py

```diff
@@ -533,3 +534,17 @@ def _overwrite_run_info(self, run_info):
         run_dir = self._get_run_dir(run_info.experiment_id, run_info.run_uuid)
         run_info_dict = _make_persisted_run_info_dict(run_info)
         write_yaml(run_dir, FileStore.META_DATA_FILE_NAME, run_info_dict, overwrite=True)
+
+    def log_batch(self, run_id, metrics, params, tags):
```

Comment:
Can this implementation be moved down to abstract store?

Reply:
Yep that's a good idea I think - doesn't hurt to provide a default implementation of `log_batch` that implementors of AbstractStore can choose to override if they'd like